ReText/window.py

```diff
@@ -211,8 +211,8 @@ def __init__(self, parent=None):
 			trig=lambda: self.insertChars('*'))
 		self.actionUnderline = self.act(self.tr('Underline'), shct=QKeySequence.Underline,
 			trig=lambda: self.insertTag('u'))
-		self.usefulTags = ('a', 'big', 'center', 'img', 's', 'small', 'span',
-			'table', 'td', 'tr', 'u')
+		self.usefulTags = ('header', 'italic', 'bold', 'numbering', 'bullets', 'image', 'link',
```

Comment:
What do you think to place everything in one place to have a consistent behaviour?
I see two options:
 - remove the Formatting submenu (I would prefer because the toolbar is more accessible)
 - remove the Tags tool and move in Formatting